tests/fixtures/parsers/typescript-parsers/new-parens.js

```diff
@@ -0,0 +1,262 @@
+"use strict";
+
+// new Storage<RootState>('state');
+
+exports.parse = () => ({
+    "type": "Program",
```

Comment:
Since this is a JS file, can you update it to follow the styleguide on the rest of the codebase? Since it's in `tests/fixtures` it looks like ESLint ignored it and the tests still passed, but I think you can automatically fix it with

```bash
bin/eslint.js --no-ignore --fix tests/fixtures/parsers/typescript-parsers/new-parens.js
```